README.md

```diff
@@ -1 +1,5 @@
-# Bucketlist
\ No newline at end of file
+# Bucketlist
```

Comment:
add more information here i.e how to set up, run tests and run the application. Simply an overview of this project